### ISSUE TITLE
Cannot change colour of focus ring on Switch elements

### ISSUE THREAD

**Issue by JackHumphries9** [chakra-ui/chakra-ui#6816]
### Description

You can change the focus ring color on inputs but not switches

### Link to Reproduction

https://codesandbox.io/s/eager-beaver-11xs8i

### Steps to reproduce

Add a switch and discover there is no `focusBorderColor` prop and then using `_focus` fails too.

### Chakra UI Version

2.3.4

### Browser

Google Chrome 106 (arm64)

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by clement-v8** [comment]
because the switch has no border.
what do you want exactly?

**Comment by JackHumphries9** [comment]
Basically when you focus on input components they have a focus ring which shows which component is in focus and for other components (i.e. input fields and selects) you can change this colour but you can't seem to do it with the Switch.

**Comment by clement-v8** [comment]
@JackHumphries9 Yeah we can not config the color when passing the props into the Switch. But if you want to have a border when focusing on the switch box, you can config the Switch in the theme like:

```javascript
baseStyle: {
    track: {
      _checked: {
        bg: 'green.50',
      },
      _focus: {
        boxShadow: '0 0 0 3px rgba(66, 153, 225, 0.6)',
      },
    },
  },
```
